lib/Value/WeBWorK.pm

```diff
@@ -73,6 +73,11 @@ sub Value::Formula::PGseedRandom {
 }
 sub Value::Formula::PGgetRandom {shift->{PGrandom}->random(@_)}
 
+sub Value::maketext {
+	my $envir =eval('$main::rh_envir');
```

Comment:
We should hold off accepting this pull request until we decide how to handle localization within MathObjects.  As Davide points out in  https://github.com/openwebwork/pg/pull/300   there may be a better way.